site-content/source/modules/ROOT/pages/development/ide.adoc

```diff
@@ -84,6 +110,12 @@ You should be able to:
 * Study Cassandra code style
 * Inspections
 
+[TIP]
+.Hint
+====
+If you wish to work with older Cassandra versions, see our https://cwiki.apache.org/confluence/display/CASSANDRA2/RunningCassandraInIDEA[wiki] for instructions.
+====
+
 === Opening Cassandra in Apache NetBeans
 
 https://netbeans.apache.org/[Apache NetBeans] is an older open source Java IDE,
```

Comment:
```suggestion
https://netbeans.apache.org/[Apache NetBeans] is the elder of the open sourced java IDEs,
```

would you mind returning this to how it was written. I know it isn't part of your changes, but it was recently changed with the new website and lost the nuance that I wrote it with ;) 
see here for the original version: https://cassandra.apache.org/doc/4.0-alpha1/development/ide.html#opening-cassandra-in-apache-netbeans